lib/src/spring_widget.dart

```diff
@@ -189,7 +192,7 @@ class _RenderAnimatedTo extends RenderProxyBox {
   }
 
   set description(SpringDescription value) {
-    _controller.spring = value;
+    _controller.motion = Motion.customSpring(value);
```

Comment:
Depending on the desired behavior, you might want to do this:

```suggestion
    _controller.motion = SpringMotion(value, snapToEnd: true);
```

This will ensure that the animation settles at the exact target value, but it might introduce a small jump at the very end. 

The version of `springster` that you used before did not have this behavior, so if you keep it the way you have it, everything will stay the same as it was before.

In a future version of motor, I will default this value to true, as having it set to false can introduce some unpredictable behavior sometimes. If you want me to elaborate on my reasoning for that I can do that, but I think for `animated_to` it's not that important.

Reply:
I guess I have to compare the behavior first, but as long as reading your comment, `animated_to` should use `snapToEnd: true` as it may be more important for package users to stop at the exact position they configure as their widget structure.

Reply:
I've compared and didn't find any problem with jumping at the end, so I've fixed using `snapToEnd: true` as you suggested. Thanks.